.gitlab/functional_test/security_agent.yml

```diff
@@ -123,7 +115,7 @@ kitchen_test_security_agent_amazonlinux_x64:
 
 kitchen_test_security_agent_x64_ec2:
```

Comment:
This one will be manual on all PRs no matter the changes, while the others runs automatically on security agent changes. Is it expected?
Otherwise LGTM

Reply:
Yes it's expected, and is already the case today. Otherwise it costs way too much